### ISSUE TITLE
Add accessibility to section 5

### ISSUE THREAD

**Issue by LJWatson** [w3ctag/design-principles#49]
Accessibility should be mentioned alongside security and privacy in section 5. This <a href="http://w3c.github.io/pfwg/wtag/checklist.html">checklist</a> may be worth referencing (though it's still a work in progress).

**Comment by dbaron** [comment]
Fixed in #59, though it didn't get marked automatically due to the lack of a space before the #.
